project/Dependencies.scala

```diff
@@ -2,7 +2,7 @@ import sbt._
 
 object Dependencies {
   lazy val lenthallV = "0.18-e690fc2-SNAPSHOT"
-  lazy val wdl4sV = "0.4"
+  lazy val wdl4sV = "0.5-574ae20-SNAPSHOT"
```

Comment:
I was asking this since we were using released version previously and not snapshots.